packages/opentelemetry-semantic-conventions-ai/opentelemetry/semconv_ai/__init__.py

```diff
@@ -29,6 +29,8 @@ class Meters:
     LLM_WATSONX_COMPLETIONS_RESPONSES = "llm.watsonx.completions.responses"
     LLM_WATSONX_COMPLETIONS_TOKENS = "llm.watsonx.completions.tokens"
 
+    LLM_BEDROCK_COMPLETIONS_EXCEPTIONS = "llm.bedrock.completions.exceptions"
```

Comment:
Is this required?

Reply:
It's an open issue. Actually, I think it's better to use a common name, for example `llm.completions.exceptions` for all ai system to count exceptions. If yes, I would like to use another PR to fix all related code.

Reply:
@jinsongo if we're doing this here I'd try to use the new genAI conventions - https://opentelemetry.io/docs/specs/semconv/gen-ai/gen-ai-metrics
I think they're already released so you can even use them directly from the original otel semconv package

Reply:
(in this case - it means exceptions are counted in the duration metric)

Reply:
@nirga I cannot find an appropriate metric name for the exceptions counter of Bedrock. As you know, version 0.4.1 of opentelemetry-semantic-conventions-ai does not include `LLM_BEDROCK_COMPLETIONS_EXCEPTIONS` although it's already included in this current PR commits, which causes the build to fail when I use it in the Bedrock instrumentation code. As you know, that's the same approach for OpenAI, Authropic and Watsonx.

Reply:
@jinsongo this should be in the standard `opentelemetry-semantic-conventions` package. And you should count exceptions on the duration metrics as specified in the spec.

Reply:
@jinsongo If this parameter is not required, I think we can ignore this for now, and we can fix this in future as a consolidation for semantic convention.

Reply:
@gyliu513 I have removed `LLM_BEDROCK_COMPLETIONS_EXCEPTIONS` from `semconv_ai`, and added a `TODO` comment about fixing in future as a consolidation for semantic convention.